lldb/source/Plugins/Language/Swift/SwiftFormatters.cpp

```diff
@@ -1530,7 +1535,8 @@ lldb_private::formatters::swift::EnumSyntheticFrontEnd::Update() {
   m_child_index = UINT32_MAX;
   m_exe_ctx_ref = m_backend.GetExecutionContextRef();
   m_element_name.SetCString(m_backend.GetValueAsCString());
-  m_child_index = m_backend.GetIndexOfChildWithName(m_element_name);
```

Comment:
llvm::Expected's destructor expects the error to have been consumed if the error exists.
The current implementation would cause a crash if we were to reach the error path.

Now this method has no error-returning mechanism (for now), so here you will have to resort to a function that just consumes the error and doesn't do anything. See `llvm::consumeError` in https://llvm.org/doxygen/llvm_2Support_2Error_8h.html

So, for example, you might be able to do:
```
if (!index_or_err)
   llvm::consumeError(index_or_err.takeError())
else
   m_child_index = *index_or_err;
```

(note that the value is already initialized to int_max in a previous line